controllers/places.js

```diff
@@ -252,3 +269,4 @@ module.exports.createPlace = createPlaces;
 module.exports.getPlace = getPlace;
 module.exports.updatePlace = updatePlace;
 module.exports.getOrCreatePlace = getOrCreatePlace;
+module.exports.minify = minify;
```

Comment:
Are we still doing the export using `_` (underscore prefix) if it's a private function thing (only exported for testing).